src/mongocxx/options/find.hpp

```diff
@@ -75,6 +75,28 @@ class MONGOCXX_API find {
     const stdx::optional<std::int32_t>& batch_size() const;
 
     ///
+    /// Sets the collation for this operation.
+    ///
+    /// @param collation
+    ///   The new collation.
+    ///
+    /// @see
+    ///   https://docs.mongodb.com/manual/release-notes/3.3-dev-series-collation/#collation-option
+    ///
+    find& collation(bsoncxx::document::view_or_value collation);
+
+    ///
+    /// Retrieves the current collation for this operation.
+    ///
+    /// @return
+    ///   The current collation.
+    ///
+    /// @see
+    ///   https://docs.mongodb.com/manual/release-notes/3.3-dev-series-collation/#collation-option
```

Comment:
Do we need a ticket to go through and update all references to 'dev' docs before release?